README.md

```diff
@@ -1,22 +1,111 @@
+
+## Requirements
+
+Download the latest version of [Xcode](https://developer.apple.com/xcode/). Xcode 11 and above comes with a [built-in Swift Package Manager](https://developer.apple.com/documentation/xcode/adding-package-dependencies-to-your-app).
+
+For more information on Swift Package Manager, open [Swift official documentation](https://swift.org/package-manager/)
+
+## Project structure
+
+`Package.swift` is the [main package manifest](https://developer.apple.com/documentation/packagedescription) that defines updated configurations to the Rokt iOS SDK package as required by the Swift Package Manager.
+
+## How to install
+
+To install for iOS development:
 
 On Xcode: 
-* Go to File > Swift Packages > Add Package Dependency
-* Add `https://github.com/ROKT/rokt-sdk-ios.git`
-* Select *Up to Next Major* with *3.9.0*
+* Go to File > Add Packages
+* Enter Package URL `https://github.com/ROKT/rokt-sdk-ios.git`
+* Select *Up to Next Major* with *3.10.0*
 
-or add below code to the `dependencies` part of `Package.swift`.
+Alternatively add below code to the `dependencies` part of `Package.swift`.
 ```swift
 dependencies: [
-    .package(url: "https://github.com/ROKT/rokt-sdk-ios.git", .upToNextMajor(from: "3.9.0"))
+    .package(url: "https://github.com/ROKT/rokt-sdk-ios.git", .upToNextMajor(from: "3.10.0"))
 ]
 ```
 
-For more information please visit [official docs](https://docs.rokt.com/docs/developers/integration-guides/ios/overview)
+### Note on Objective-C integration
+
+The Rokt SDK for iOS is implemented using Swift. If you are using Objective-C, you need to import the bridging header file from the framework into the .h/.m file.
+```objective-c
+#import <Rokt_Widget/Rokt_Widget-Swift.h>
+```
+If you are having trouble installing and are receiving an error saying that the SWIFT_VERSION is not defined, please add a user-defined variable `SWIFT_VERSION`. This variable should be set to `5` for iOS SDK versions 2.0 and above. This variable should be set to `4.2` for iOS SDK version 1.2.1.
+
+## How to test integration
+
+The following steps test an overlay placement - only 2 explicit calls, `init` and `execute`, are needed.
+
+### 1. Initialise module for testing
+
+```swift
+import Rokt_Widget
+
+func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
+    Rokt.initWith(roktTagId: "your_tag_id_here")
+    return true
+}
+```
+
+Contact Rokt for the Rokt Account ID associated with your account and enter your unique Rokt Account ID as the `roktTagId`.
+
+### 2. Execute (overlay placement)
+
+Firstly define in `ViewController` e.g.
+
+```swift
+func showWidget() {
+    let attributes = ["email": "j.smith1674613477652@example.com",
+            "firstname": "Jenny",
+            "lastname": "Smith",
+            "mobile": "(555)867-5309",
+            "postcode": "90210",
+            "country": "US",
+            "sandbox": "true"]
```

Comment:
Sandbox = true would force the SDK to use the sandbox environment and they need to be aware of it. We have this part separately in https://docs.rokt.com/developers/integration-guides/ios/reference/sandbox-integration